docker/Dockerfile

```diff
@@ -36,17 +40,20 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
     curl \
     smartmontools \
     tzdata \
-    && update-ca-certificates \
-    &&  case ${TARGETARCH} in \
-            "amd64")  S6_ARCH=amd64  ;; \
-            "arm64")  S6_ARCH=aarch64  ;; \
-        esac \
+    && update-ca-certificates
+RUN case ${TARGETARCH} in \
```

Comment:
is there a reason you wanted to split up this RUN into multiple steps? 
Is this a leftover from debugging?